lib/elixir/test/elixir/kernel/import_test.exs

```diff
@@ -12,6 +12,20 @@ defmodule Kernel.ImportTest do
     end
   end
 
+  defmodule SampleA do
```

Comment:
Typically we avoid defining new modules in tests because defining tons of modules slow downs the test suite. If you could rewrite it maybe using `Elixir.{List, String}` or such, it would be better.